ru.js

```diff
@@ -61,7 +61,7 @@ generic.accuracy Точность
 player.waiting Ожидание игроков
 player.died Вы умерли
 player.killed Убиты
-player.killstreak {0} on a {1} Kill Streak
+player.killstreak {0} сделал {1} убийств подряд!
```

Comment:
Can you revert this one back to English please

Reply:
@@ -61.7 +61.7 @@ generic.accuracy Accuracy
player.waiting Waiting for players
player.died You are dead
player.killed Killed
player.killstreak {0} on a {1} Kill Streak
player.killstreak {0} made {1} kills in a row!